components/atom/switch/src/index.js

```diff
@@ -1,74 +1,68 @@
-import React, {Component} from 'react'
+import React, {useState} from 'react'
 import PropTypes from 'prop-types'
 import {ToggleSwitchTypeRender} from './SwitchType/toggle'
 import {SingleSwitchTypeRender} from './SwitchType/single'
 import {SIZES, TYPES, SUPPORTED_KEYS} from './config'
 
-class AtomSwitch extends Component {
-  state = {
-    isToggle: this.props.initialValue,
-    isFocus: false,
-    isClick: false
-  }
+const AtomSwitch = props => {
+  const {initialValue, disabled, onToggle: _onToggle, value, type} = props
```

Comment:
I hate commenting about naming (sorry 🙇 ) but maybe `_onToggle` is a bit weird to read. What about `onToggleCallback`?

Reply:
👍 Much better. Changed to `onToggleCallback `!